index.html

```diff
@@ -4,19 +4,26 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Library App</title>
   </head>
   <body>
     <h1 class="text-4xl">Awesome Books</h1>
     <div class="books-list-container" id="bookslist">
       <p id="bookstatus"></p>
     </div>
-    <hr><br>
+    <br />
+    <br />
+    <hr />
```

Comment:
- [ ] OPTIONAL
It will be appropriate to use CSS spacing (padding and margins) to add spacing to your elements other than using the `br` tag